### ISSUE TITLE
Add Quickstart Guide

### ISSUE THREAD

**Issue by davidekete** [knex/knex#5970]
New users of Knex have a hard time starting out since the documentation has no quickstart guide. To configure and run queries with Knex, you have to go through most of the documentation, which is very time-consuming. I propose adding a quick start guide for various database drivers as a section in the documentation. I can help with that if it is something the core maintainers are open to.

**Comment by mercmobily** [comment]
You're right that the existing documentation at https://knexjs.org/guide/ is comprehensive but not optimised for new users who just want to get started quickly. It covers many advanced configuration options before showing a simple working example.

A proper quickstart guide would be valuable. It should provide:

1. Minimal installation steps for the main databases (PostgreSQL, MySQL, SQLite3)
2. A complete, runnable example from npm install to first query
3. Common troubleshooting for first-time setup
4. Clear pointers to the comprehensive guide for deeper learning

This connects to broader documentation improvements being tracked. Issue #6266 discusses improving community engagement and notes that better onboarding documentation reduces support burden. Multiple closed issues show users struggling with basic setup questions that a quickstart would address.
